test/screenshotter/ss_data.yaml

```diff
@@ -33,7 +33,7 @@ Alignedat: |
         3a &= 3 & 17b &= 34 &\quad 400c &= 1200
     \end{alignedat}
 Arrays: |
-    \left(\begin{array}{|rl|c||}
+    \left(\begin{array}{|rl:c}
```

Comment:
I think we still want to test (the space between) a double vertical bar, if we can fit it in the test...

Reply:
Woops, okay yeah I'll put that back.